src/test/java/net/smartcosmos/usermanagement/role/persistence/RolePersistenceServiceTest.java

```diff
@@ -129,6 +129,58 @@ public void thatUpdateRoleSucceeds() {
     }
 
     @Test
+    public void thatUpdateRoleSucceedsWithSameName() {
+
+        final String roleName = "updateTestRole2";
```

Comment:
This and any other tests should be mocking the back end datastore (database).  If you want to test the database really works then write Integration Tests.